src/phy/tracer.rs

```diff
@@ -148,15 +147,56 @@ impl<Tx: phy::TxToken> phy::TxToken for TxToken<Tx> {
     }
 }
 
-pub struct Packet<'a> {
+/// Packet which is being traced by [Tracer](struct.Tracer.html) device.
+pub struct TracerPacket<'a> {
     buffer: &'a [u8],
     medium: Medium,
-    prefix: &'static str,
+    direction: TracerDirection,
```

Comment:
I think all of these can be `pub` fields. Also, the struct as a whole should have `#[derive(Debug, Clone)]` at least (probably `Copy` too)

Reply:
Updated interface of `TracerPkt` to expose all fields publicly, removed now redundant constructor and accessor fields. Added `Debug/Clone/Copy` auto-derives.